hack/scripts/netperf.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+# find the nodes with netperf pods and assign test vars
+node_found=0
+for node in $(kubectl get nodes -o name);
```

Comment:
We can limit this result to three if we use the label selector `netperf=true`?

Reply:
yes, in label-nodes.sh I cut off the `netperf=true` labeling once 3 nodes have it